src/txmempool.cpp

```diff
@@ -1055,11 +1055,10 @@ void CTxMemPool::TrimToSize(size_t sizelimit, std::vector<COutPoint>* pvNoSpends
     }
 }
 
```

Comment:
nit: You don't have to do this, but I think it would be better if this commit was self contained, as now the clients of `TransactionWithinChainLimit` are not compiling anymore. To very your change is valid we need to check other commits later. This make it harder to review.